cluster-controller/src/test/java/io/strimzi/controller/cluster/ResourceUtils.java

```diff
@@ -72,7 +73,72 @@ public static ConfigMap createConfigMap(String clusterCmNamespace, String cluste
                 .withNewMetadata()
                 .withName(clusterCmName)
                 .withNamespace(clusterCmNamespace)
-                .withLabels(singletonMap("strimzi.io/kind", "kafka-cluster"))
+                .withLabels(labels("strimzi.io/kind", "cluster", "strimzi.io/type", "kafka"))
+                .endMetadata()
+                .withData(cmData)
+                .build();
+    }
+
+
+    /**
+     * Generate ConfigMap for Kafka Conect S2I cluster
+     *
+     * @param clusterCmNamespace
```

Comment:
Can we fill in the doc for these params, or otherwise delete them. Having them present but undocumented just seems pointless, imho.

Reply:
Copy paste from your tests :-)